frappe/core/doctype/communication/email.py

```diff
@@ -247,7 +247,7 @@ def add_attachments(name: str, attachments: Iterable[str | dict]) -> None:
 			{
 				"attached_to_doctype": "Communication",
 				"attached_to_name": name,
-				"folder": "Home/Attachments",
+				"folder": frappe.db.get_value("File", {"is_attachments_folder": 1}),
 			}
```

Comment:
Maybe add `bootinfo.attachments_folder` to frappe/boot.py?

Reply:
I like this idea. It could help a big refactor

Reply:
I like this idea. Now there are two paths going forward:
* refactor [everything that uses a hardcoded `Home/Attachments` reference](https://github.com/search?q=org%3Afrappe+Home%2FAttachments&type=code) to use frappe.boot.attachments_folder
* stop translating Home and Home/Attachments on setup, and write a patch to convert existing translated folders